### DISCUSSION TITLE
Existe outro aplicativo para grids como o Jeet?

### DISCUSSION THREAD

**Discussion by Marcelosilva10** [frontendbr/forum#199] [Perguntas e Respostas]
Bom esses dias estava conversando com meu amigo, e ele me indicou o Jeet não usei ainda, tem outro que você podem indicar?

**Comment by vitor-mendrone-deel** [comment]
Existem alguns sistemas de grids muito bons por ai. 

O [Lost](https://github.com/peterramsing/lost) atualmente me parece o melhor, é do mesmo criador do Jeet e tem um poder de fogo considerável, sem contar o suporte ao Flexbox. (O @neysimoes e o @LFeh podem falar mais sobre)

O @afonsopacifer tem um framework próprio chamado [Flex grid Framework](http://flexgridframework.com/), baseado em flexbox, super fácil de usar (já testei e curti a forma como as coisas funcionam nele) 

O [Kouto Swiss](http://kouto-swiss.io/docs.html) é uma outra alternativa que já testei e curti bastante pela simplicidade.

Os 3 que citei são ferramentas do Stylus, o legal é testar cada um deles pra entender qual se encaixa melhor a sua necessidade. 

De qualquer forma, minha escolha hoje para iniciar um novo projeto seria sem dúvidas o Lost!

**Comment by douglascorrea** [comment]
@Marcelosilva10 se vc prefere sass vc tem o [Suzy](http://susy.oddbird.net/) que ainda mais poderoso que o Jeet e bem mais documentado.

A nova versão do [Foundation](http://foundation.zurb.com/) também vem com um grid flexível incluindo uma versão baseada em Flexbox

**Comment by felipefialho** [comment]
Vai de [Lost](https://github.com/peterramsing/lost), não tem erro.
